src/main/java/nl/topicus/mssql2monetdb/CopyToolConfig.java

```diff
@@ -731,6 +733,17 @@ else if (key.equals("port"))
 		
 		return sourceDatabases;
 	}
+	
+	private String readQueryFile(String filePath)
+	{
+		try {
+			return new String(Files.readAllBytes(Paths.get(filePath)));
+		} catch (IOException e) {
+			LOG.error("Unable to read query file {} due to: {}", filePath, e.getMessage());
+		}
+		
+		return null;
```

Comment:
Denk niet dat je `null` in het geval van en exceptie wil teruggeven, maar de exceptie weer wilt opgooien? Lijkt me niet dat je door kunt gaan als het lezen mislukt is.